src/particles/distribution/T2Z.H

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2021 Chad Mitchell, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_T2Z_H
+#define IMPACTX_T2Z_H
+
+#include "particles/ImpactXParticleContainer.H"
+
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
```

Comment:
I would add an extra namespace here, to keep them in sync with the directory depth:
```suggestion
namespace impactx
{
namespace distribution
{
```